control/resetSettings.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
I'd call it defaultSettings since it should not just be the settings for the reset but for all the service kcontrold

Reply:
It is been used just for the reset, since the control does not need any parameters on its creation.